### PR TITLE
Stanford ME113 Spring 2013 Project

### PR DESCRIPTION
**Pull request by embirico**
Hi,

I'm submitting the documentation for the project we (4 Stanford students) did with Dave Evans and the Ford SVL. (It's a little late!) Our current contact with Ford is Sudipto Aich.

Happy to answer any questions or address any comments.

Thanks and best,
Alex

### CODE REVIEW

**Comment by peplin** [comment]
Awesome, thanks Alex - this looks great. I'll get it merged today or tomorrow.

File: projects/stanford-me113-spring-2013/shape-retaining-mount.mkd [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+<div class="page-header">
+    <h2>Appendix</h2>
+</div>
+
+Four balloon prototype
+----------------------
+![Four balloon prototype](/projects/images/stanford-me113-spring-2013/srm_balloon_prototype.jpg)
+
+Bonus picture: glove SRM
```
Comment (sudo-a13) [inline projects/stanford-me113-spring-2013/shape-retaining-mount.mkd]: Alex, thanks again for pushing this. I know this is kind of a bonus picture, but can you write up a sentence on the bonus picture "Glove SRM", may be obvious may be not, would not hurt to have it

Comment (embirico) [inline projects/stanford-me113-spring-2013/shape-retaining-mount.mkd]: Yep I'll ask Jacob for one and add the commit when he gets back to me.

File: projects/stanford-me113-spring-2013/index.mkd [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+[3dpmLink]: /projects/stanford-me113-spring-2013/3d-printed-tablet-mount.html
+
+<div class="page-header">
+    <h1>Stanford Senior Design Project</h1>
+</div>
+
+**GitHub Repository:**
+[here](https://github.com/embirico/stanford-me113-spring-2013)
+
+The following work was undertaken (enjoyed) as a senior capstone design project
```
Comment (sudo-a13) [inline projects/stanford-me113-spring-2013/index.mkd:17]: I would also add the names of your team member, and  may be the link of your individual contact/website if you want, but at least the team members.

Comment (embirico) [inline projects/stanford-me113-spring-2013/index.mkd:17]: Sounds good.

**Comment by embirico** [comment]
Ok @sudipto13 I've addressed your two comments :)

**Comment by sudo-a13** [comment]
Thanks Alex!!

**Pull request merged**